### ISSUE TITLE
[Screen Reader - Howdy - Delete]: Narrator is not narrating "Script deleted" dialog information after selecting delete button.

### ISSUE THREAD

**Issue by v-gasikk** [howdyai/howdy-website#16]
**User Experience:**
Users who rely on screen readers will affect to navigate, if Narrator is not narrating "Script deleted" dialog information after selecting delete button.

**Test Environment:**
OS: Win 10 Version 1809 (OS Build 17763.316)
URL: https://howdy.ai/
Browser: Edge
Screen Reader: Narrator

**Repro Steps:**
Step 1: Open the above URL in Edge browser.
Step 2: Open this URL: https://hcl-szz1925.howdy.ai/dashboard in edge.
Step 3: “Dashboard” page gets opened.
Step 4: Navigate to “Scripts” tab and select it.
Step 5: Navigate to "Delete" button and select it.
Step 6: "Delete check in" dialog gets opened.
Step 7: Navigate to "Delete" button in "Delete check in" dialog and select it.
Step 8: Observe whether Narrator is narrating Script deleted information or not.

**Actual Result:**
Narrator is not narrating "Script deleted" dialog information after selecting delete button.
Ex: When user selecting "delete" button, Narrator is not narrating any information.

**Expected Result:**
Narrator should narrate "Script deleted" information after selecting delete button.
Ex: When selecting "delete" button, Narrator should narrate "Script deleted" info and focus should be on "Ok" button.

**MAS Reference Link:**
[https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}](url)

[MAS1.3.1_Howdy_Script deleted info.zip](https://github.com/howdyai/howdy-website/files/2973754/MAS1.3.1_Howdy_Script.deleted.info.zip)
![MAS1 3 1_Howdy_Script deleted info](https://user-images.githubusercontent.com/48272005/54473717-aca25e00-4801-11e9-8dc4-12d830e04d25.png)

**Comment by benbrown** [comment]
@awalia13 I am not sure how to trigger the screen reader to read the modal popup.  Can you suggest a solution?

**Comment by benbrown** [comment]
add an aria-live="true" tag

**Comment by benbrown** [comment]
added aria-live="polite"

**Comment by v-gasikk** [comment]
Verified. Issue is Not Fixed. Hence Re-activating the bug.
Narrator is not narrating the "Script deleted" dialog information after selecting "delete" button.
[MAS1.3.1_Script deleted_Not Fixed.zip](https://github.com/howdyai/howdy-website/files/3038230/MAS1.3.1_Script.deleted_Not.Fixed.zip)

**Comment by benbrown** [comment]
I re-implemented this per @awalia13's guideance, and it seems to work with Voiceover on my Mac. Please reverify.

**Comment by awalia13** [comment]
Closed as it is working fine.
